### PR TITLE
Add configurable contract addresses and asset symbol mapping

### PR DESCRIPTION
**Pull request by konstantinzolotarev**
This obsoletes #176.

### CODE REVIEW

File: internal/config/gofer/origin.go [hunk 1]
```diff
@@ -23,98 +23,110 @@ import (
 	"github.com/makerdao/oracle-suite/pkg/gofer/origins"
 )
 
-type apiKeyParams struct {
-	APIKey string `json:"apiKey"`
+type originParams struct {
+	APIKey        string                    `json:"apiKey"`
+	Contracts     origins.ContractAddresses `json:"contracts"`
+	SymbolAliases origins.SymbolAliases     `json:"symbolAliases"`
```
Comment (MDobak) [inline internal/config/gofer/origin.go:29]: I think this should be handled by every origin separately.

**Review by MDobak — CHANGES_REQUESTED** [review]

File: internal/config/gofer/origin.go (hunk 1 above)
Comment (konstantinzolotarev) [reply internal/config/gofer/origin.go:29]: This you mean `APIKey` only or all options ?

**Comment by teghnet** [comment]
Looks good to me. Just waiting for the final version.

File: pkg/gofer/origins/upbit_test.go [hunk 2]
```diff
@@ -158,16 +159,16 @@ func (suite *UpbitSuite) TestSuccessResponse() {
...
-	pairs := []Pair{
-		{Base: "KNC", Quote: "KRW"},
-		{Base: "MANA", Quote: "KRW"},
-		{Base: "OMG", Quote: "KRW"},
-		{Base: "REP", Quote: "KRW"},
-		{Base: "SNT", Quote: "KRW"},
-		{Base: "ZRX", Quote: "KRW"},
-	}
-	testRealBatchAPICall(suite, &Upbit{Pool: query.NewHTTPWorkerPool(1)}, pairs)
+	//testRealAPICall(suite, &Upbit{Pool: query.NewHTTPWorkerPool(1)}, "ETH", "BTC")
```
Comment (teghnet) [inline pkg/gofer/origins/upbit_test.go:162]: Why this comments?

**Review by teghnet — CHANGES_REQUESTED** [review]

File: pkg/gofer/origins/upbit_test.go (hunk 2 above)
Comment (konstantinzolotarev) [reply pkg/gofer/origins/upbit_test.go]: Ouch, I uncommented all except this one. Good catch !

**Review by MDobak — APPROVED** [review]

**Review by teghnet — APPROVED** [review]

**Pull request merged**